src/BuildCast/Views/Downloads.xaml

```diff
@@ -24,7 +24,9 @@
     xmlns:views="using:BuildCast.Views"
     mc:Ignorable="d" Loaded="Downloads_Loaded"
     xmlns:helpers="using:BuildCast.Helpers"
-    >
+    xmlns:animations="using:Microsoft.Toolkit.Uwp.UI.Animations"
+    Canvas.ZIndex="1"
```

Comment:
This will not work for all pages in the app.. needs to be set / called in the onnavigatingfrom of each page that is using the defaultpagehide transition to ensure every page is "on top" at the moment the on hide animation is playing